cms/static/sass/bootstrap/_components.scss

```diff
@@ -0,0 +1,195 @@
+// Open edX: components
+// ====================
```

Comment:
@HarryRein My thinking is that this is really a task for the Educator team, so I wasn't planning on putting a lot of work into this. I wanted to give them a baseline that they could work from. I'm happy to discuss whether there's a better compromise we can reach here. At a minimum, I can create a story in the Educator project.